src/Boson_Sampling_Utilities.py

```diff
@@ -215,15 +316,15 @@ def __can_calculation_be_performed(self) -> bool:
 
         return can_calculation_be_performed
 
-    def __calculate_v_vectors(self):
+    def __calculate_v_vectors(self, input_vector: ndarray = []):
```

Comment:
Mutable assigned as a default value AND incorrect type (expected `ndarray`, got `list` instead)